### PR TITLE
Update `Gemfile.lock`

### PR DESCRIPTION
**Pull request by NachoSoto**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -338,7 +338,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  arm64-darwin-22
```
Comment (tonidero) [inline Gemfile.lock:341]: Hmm I'm not sure if this will have any issues in any of the CircleCI machines...

File: Gemfile.lock [hunk 2]
```diff
@@ -88,18 +88,18 @@ GEM
     concurrent-ruby (1.2.0)
     cork (0.3.0)
       colored2 (~> 3.1)
-    danger (9.2.0)
+    danger (8.6.1)
```
Comment (tonidero) [inline Gemfile.lock:91]: This seems like a downgrade, but I think it's probably ok, I know bumping dunger actually broke some things in Android, not sure if it's also causing any issues for you?

**Review by tonidero — APPROVED** [review]
If tests pass I think this should be fine, but I'm not sure with the Platforms change...

File: Gemfile.lock (hunk 1 above)
Comment (NachoSoto) [reply Gemfile.lock:341]: Yeah let me reverse this.

File: Gemfile.lock (hunk 2 above)
Comment (NachoSoto) [reply Gemfile.lock:91]: Yeah I was confused, but that's what I get when I do `gem install`?

Comment (tonidero) [reply Gemfile.lock:91]: Hmm I see the `Gemfile` doesn't specify the version, so it should be using the latest... Maybe you are using a cached version? But that's weird, I don't know how that happened either 😅

**Pull request closed without merging**